tests/test_filters.py

```diff
@@ -319,3 +320,82 @@ def test_filter_with_non_existing_op(self):
             batch.append(record)
         with self.assertRaisesRegexp(InvalidOperator, 'operator not valid'):
             KeyValueFilter({'options': {'keys': keys}}, meta())
+
+
+class DupeFilterTest(unittest.TestCase):
+
+    def test_filter_duplicates_with_default_key(self):
+        keys = ['8062219f00c79c88', '1859834d918981df', 'e2abb7b480edf910']
+        items = [
+            {'_key': keys[0], 'name': 'item1', 'country_code': 'es'},
+            {'_key': keys[0], 'name': 'item1', 'country_code': 'es'},
+            {'_key': keys[1], 'name': 'item2', 'country_code': 'us'},
+            {'_key': keys[1], 'name': 'item2', 'country_code': 'us'},
+            {'_key': keys[2], 'name': 'item3', 'country_code': 'uk'},
+            {'_key': keys[2], 'name': 'item3', 'country_code': 'uk'}
```

Comment:
These inputs are only testing cases when the whole object is the same, it would be nice to make the expectations about ordering a bit more clear, in case objects have same key but differ in content.

I suggest making the `name` field different for each item of the input.

That will make the expectation "use the first it finds, ignore the next ones" more clear.